eventtracking/backends/async_routing.py

```diff
@@ -0,0 +1,50 @@
+"""Route events to processors and backends"""
```

Comment:
Keeping this code in routing backend causes "Models aren't  loaded yet" error. `openedx/features/course_experience/__init__.py` file imports from `lms/djangoapps/experiments/flags.py` that imports segment tracker. And that tracker uses routing backend. Hence if we keep this backend in `routing.py` file, it will try to load models during the `__init__.py` file loading and will throw error.

Reply:
Can we move `AsyncRoutingBackend ` to routing.py?